templates/requirements.txt.j2

```diff
@@ -0,0 +1,5 @@
+docker-py{{ '' if docker_py_version == 'latest' else ('==' + docker_py_version) }}
+dockerrotate=={{ docker_rotate_version }}
+{% if docker_compose_version | bool %}
```

Comment:
That's because that filter is trying to figure out if a string says some form of "true"